test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -23,8 +23,32 @@ struct suite_context_s {
 
 static struct suite_context_s suite_context;
 
-static int packet_cmp_mem(odp_packet_t pkt, uint32_t offset,
-			  void *s, uint32_t len)
+static int packet_cmp_mem_bits(odp_packet_t pkt, uint32_t offset,
+			       uint8_t *s, uint32_t len)
+{
+	int rc = -1;
+	uint32_t len_bytes = ((len + 7) >> 3);
+	uint8_t leftover_bits = len & 0x7;
```

Comment:
A very minor comment: I think this would be more readable:
uint32_t len_bytes = (len + 7) / 8;
uint8_t leftover_bits = len % 8;

Reply:
Will update this in v4.